luigi/notifications.py

```diff
@@ -254,19 +251,15 @@ def send_email_sendgrid(sender, subject, message, recipients, image_png):
     client.send(to_send)
 
 
-def _email_disabled():
+def _email_disabled_reason():
     if email().format == 'none':
-        logger.info("Not sending email when email format is none")
-        return True
+        return "Not sending email when email format is 'none'"
     elif email().force_send:
-        return False
+        return ''
```

Comment:
Ah right, I thought `'some_string' + None` will crash, but actually such logic won't be run. I'll fix this. :)